src/main/java/org/jabref/gui/desktop/os/Windows.java

```diff
@@ -20,6 +20,7 @@
 import mslinks.ShellLink;
 import mslinks.ShellLinkException;
 import org.slf4j.LoggerFactory;
+import org.tinylog.Logger;
```

Comment:
should be org.slf4j.Logger
It's a facade, see in other classes how to use it

Reply:
I've made a new static variable `LOGGER`, as I saw in other classes, but got an error while running:
```
Messages are not initialized before accessing key: Display help on command line options
Exception in thread "main" java.lang.UnsupportedOperationException: Configuration cannot be changed after applying to tinylog
	at org.tinylog.api@2.7.0/org.tinylog.configuration.Configuration.set(Configuration.java:288)
	at java.base/java.util.Map.forEach(Map.java:733)
	at org.jabref@100.0.0/org.jabref.Launcher.addLogToDisk(Launcher.java:149)
	at org.jabref@100.0.0/org.jabref.Launcher.main(Launcher.java:66)
```

Reply:
Ah yes @koppor  Do we have to use the lazy one here? Logger.getLogger?

Reply:
Yes, Lazy one. See this class. If this is not existing, check the other classes in that package.

Reply:
I'm sorry, this was actually explicitly stated in class comment 😄.